{"Modified":"2020-09-02T14:31:00","Published":"2020-08-25T14:15:00","access":{"authentication":"NONE","complexity":"MEDIUM","vector":"NETWORK"},"assigner":"cve@mitre.org","capec":[],"cvss":7.1,"cvss-time":"2020-09-02T14:31:00","cvss-vector":"AV:N/AC:M/Au:N/C:N/I:N/A:C","cwe":"CWE-416","id":"CVE-2020-24240","impact":{"availability":"COMPLETE","confidentiality":"NONE","integrity":"NONE"},"last-modified":"2020-09-02T14:31:00","references":["https://github.com/akimd/bison/commit/be95a4fe2951374676efc9454ffee8638faaf68d","https://github.com/akimd/bison/compare/v3.7...v3.7.1","https://lists.gnu.org/r/bug-bison/2020-07/msg00051.html"],"refmap":{"misc":["https://github.com/akimd/bison/commit/be95a4fe2951374676efc9454ffee8638faaf68d","https://github.com/akimd/bison/compare/v3.7...v3.7.1","https://lists.gnu.org/r/bug-bison/2020-07/msg00051.html"]},"summary":"GNU Bison before 3.7.1 has a use-after-free in _obstack_free in lib/obstack.c (called from gram_lex) when a '\\0' byte is encountered. NOTE: there is a risk only if Bison is used with untrusted input, and the observed bug happens to cause unsafe behavior with a specific compiler/architecture. The bug report was intended to show that a crash may occur in Bison itself, not that a crash may occur in code that is generated by Bison.","vulnerable_configuration":[{"id":"cpe:2.3:a:gnu:bison:3.7:*:*:*:*:*:*:*","title":"cpe:2.3:a:gnu:bison:3.7:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:gnu:bison:3.7:*:*:*:*:*:*:*"]}